ethcore/sync/src/chain/mod.rs

```diff
@@ -938,6 +938,16 @@ impl ChainSync {
 				self.state = SyncState::Blocks;
 				self.continue_sync(io);
 			},
+			SyncState::SnapshotData => {
+				match io.snapshot_service().status() {
+					RestorationStatus::Inactive |
```

Comment:
IMHO the line breaks are unnecessary, I'd also move the `match to line above so that it looks like this:
```
SyncState::SnapshotData => match io.snapshot_service().status() {
  RestorationStatus::Inactive | RestorationStatus::Failed => {
    ...
  },
  RestorationStatus::Initializing { .. } | RestorationStatus::Ongoing { .. } => (),
}